airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/model/AlterationSummary.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.clickhouse_v2.model
+
+data class AlterationSummary(
+    val added: Map<String, String>,
+    val modified: Map<String, String>, // old type -> new type
+    val deleted: Set<String>
```

Comment:
Nit: codifying the meaning of these strings via structs with names might read better.